koji_containerbuild/plugins/builder_containerbuild.py

```diff
@@ -406,11 +409,16 @@ def handler(self, src, target_info, arch, scratch=False,
         build_id = build_response.get_build_name()
         self.logger.debug("OSBS build id: %r", build_id)
 
-        self.logger.debug("Waiting for osbs build_id: %s to be scheduled.",
-                          build_id)
-        # we need to wait for kubelet to schedule the build, otherwise it's 500
-        self.osbs().wait_for_build_to_get_scheduled(build_id)
```

Comment:
I wonder if Origin behaves the same way. Don't want to tie this to OSE specifically.